scripts/build_seawater_temperature_copernicusmarine.py

```diff
@@ -0,0 +1,149 @@
+# SPDX-FileCopyrightText: Contributors to PyPSA-Eur <https://github.com/pypsa/pypsa-eur>
+#
+# SPDX-License-Identifier: MIT
+"""
+Build seawater temperature data from Copernicus Marine Service.
+
+This script downloads historical seawater temperature data for use in sea water
+heat pump calculations. It retrieves potential temperature (thetao) data from
+the global ocean physics reanalysis dataset at daily resolution.
+
+The data covers European coastal areas at a spatial resolution of 0.083° and
+includes near-surface depths (depth range set in config) suitable for heat pump applications.
+
+Relevant Settings
+-----------------
+
+.. code:: yaml
+
+    copernicusmarine:
+        dataset_id: ID of the dataset to download from copernicusmarine package
+        variables: Potential temperature [°C]
+        depth: Near-surface depth for heat pumps [m]
+
+    # Uses cutout wildcard from Snakemake rule
+
+Inputs
+------
+- None (downloads from Copernicus Marine Service)
+
+Outputs
+-------
+- `data/seawater_temperature_copernicusmarine_{cutout}.nc`: NetCDF file containing seawater temperature data
+
+Notes
+-----
+Requires Copernicus Marine Service credentials configured via copernicusmarine package.
+See https://marine.copernicus.eu/ for account setup and API access.
```

Comment:
Does that mean that by default the workflow will not work and someone has to figure out how to set up the API access before running the model @amos-schledorn ?
That's not what we usually do for default. We usually setup the model to be executable without the need to configure any API access credentials...

Reply:
@euronion Yes, you need API access for the retrieval. Seawater heat pumps are treated as a niche feature for district-heating-focussed studies and are disabled by default. The test workflow is handled through the bundle data.

Reply:
The setup instructions for the API are these, right? https://help.marine.copernicus.eu/en/articles/8185007-copernicus-marine-toolbox-credentials-configuration#h_f60e2fce95

(I had to search for a couple of minutes to find them, so best to add a direct link to them as well.)
```suggestion
Requires Copernicus Marine Service credentials configured via copernicusmarine package.
For account setup and API access see this [documentation article](https://help.marine.copernicus.eu/en/articles/8185007-copernicus-marine-toolbox-credentials-configuration#h_f60e2fce95) on the [Copernicus Marine website](https://marine.copernicus.eu/).
```